.gitmodules

```diff
@@ -1,5 +1,5 @@
 [submodule "docker/optimism"]
-	path = docker/optimism
+	path = docker/optimism/optimism
```

Comment:
why change this?

Reply:
> why change this?

This makes it consistent with the arbitrum stuff and also leaves the room for optimism-specific scripts in `docker/optimism`, like the one we already have, `optimism.sh`.

I also thought about moving all that stuff into e.g. `${ROOT}/integrations/` or `${ROOT}/rollups/` or something similar, because it does not have anything to do with our docker setup, as the toplevel dir name may suggest.